src/config/ConfigManager.cpp

```diff
@@ -2184,6 +2185,8 @@ std::optional<std::string> CConfigManager::handleBind(const std::string& command
             ignoreMods = true;
         } else if (arg == 's') {
             multiKey = true;
+		} else if (arg == 'o') {
+			longPress = true;
```

Comment:
requires a check for incompatibilities. for example, long press + release makes no sense.

further down you'll see some checks that return an error